app/controllers/orders/order_types.js

```diff
@@ -1,3 +1,54 @@
 import detail from "./detail";
+import Ember  from 'ember';
+import _ from 'lodash';
 
-export default detail.extend();
+export default detail.extend({
+
+  logisticDataRows: Ember.computed('model', function () {
+    const order = this.get('model');
+    return [
+      // Schedule
+      {
+        label: this.get('i18n').t('order_details.logistics.scheduled'),
```

Comment:
This seems like repetition of same... Can we create a function which will take params and returns hash depending on param??

Reply:
we can, but it'd end up similar, calling the function again and again.

e.g
```
makeRow(
        this.get('i18n').t('order_details.logistics.type'),
        order.get('bookingTypeLabel'),
        function action() {},
        'tv',
        'type'
)
```

I'm okay with that, but i don't find it significantly better.
It's just data after all.
We could move everything to the `hbs` and not have a foreach, but i think it clutters the template too much, this is long but very understandable

Reply:
I could make separate functions, e.g `makeVehicleRow()` , `makeScheduleRow()`, and return 
```
[ makeVehicleRow(), makeScheduleRow() ]
```

but then again, is it really a better alternative, i'm not so sure tbh